packs/data/spells.db/lay-on-hands.json

```diff
@@ -38,19 +31,76 @@
         "hasCounteractCheck": {
             "value": false
         },
-        "heightening": {
-            "damage": {
-                "0": "6"
-            },
-            "interval": 1,
-            "type": "interval"
-        },
         "level": {
             "value": 1
         },
         "materials": {
             "value": ""
         },
+        "overlays": {
+            "a33QUFoKgoOprovO": {
+                "_id": "a33QUFoKgoOprovO",
+                "name": "Lay on Hands (Vs. Undead)",
+                "overlayType": "override",
+                "sort": 2,
+                "system": {
+                    "damage": {
+                        "value": {
+                            "37YW4ZGhxx7Y2mdI": {
+                                "applyMod": false,
+                                "type": {
+                                    "categories": [],
+                                    "subtype": "",
+                                    "value": "positive"
+                                },
+                                "value": "1d6"
+                            }
+                        }
+                    },
+                    "heightening": {
+                        "damage": {
+                            "37YW4ZGhxx7Y2mdI": "1d6"
+                        },
+                        "interval": 1,
+                        "type": "interval"
+                    },
+                    "save": {
+                        "basic": "basic"
+                    }
```

Comment:
```suggestion
                    "save": {
                        "basic": "basic",
                        "value": "fortitude"
                    }
```

Reply:
Weird, it's definitely there in Foundry.

Reply:
Ah, found it. It was still on the parent.